src/update-audio-settings.js

```diff
@@ -21,23 +22,37 @@ function applySettings(audio, settings) {
     audio.setRolloffFactor(settings.rolloffFactor);
     audio.setRefDistance(settings.refDistance);
     audio.setMaxDistance(settings.maxDistance);
+    audio.panner.panningModel = settings.panningModel;
     audio.panner.coneInnerAngle = settings.coneInnerAngle;
     audio.panner.coneOuterAngle = settings.coneOuterAngle;
     audio.panner.coneOuterGain = settings.coneOuterGain;
   }
   audio.gain.gain.setTargetAtTime(settings.gain, audio.context.currentTime, 0.1);
 }
 
+// Return panning model depending on the preferences
+export function getPanningModelType() {
+  return APP.store.state.preferences.audioPanningQuality === "Low"
+    ? PanningModelType.EqualPower
+    : PanningModelType.HRTF;
+}
+
 export function getCurrentAudioSettings(el) {
   const sourceType = APP.sourceType.get(el);
   const defaults = defaultSettingsForSourceType.get(sourceType);
   const sceneOverrides = APP.sceneAudioDefaults.get(sourceType);
   const audioOverrides = APP.audioOverrides.get(el);
   const audioDebugPanelOverrides = APP.audioDebugPanelOverrides.get(sourceType);
   const zoneSettings = APP.zoneOverrides.get(el);
-  const preferencesOverrides = APP.store.state.preferences.disableLeftRightPanning
-    ? { audioType: AudioType.Stereo }
-    : {};
+
+  const preferencesOverrides = {
+    panningModel: getPanningModelType()
```

Comment:
If the user does not specify a preference, we should not add `panningModel` to `preferencesOverrides`. 

If the user does not have a preference, their preference is `null`:

```js
function getPanningModelType() {
  return APP.store.state.preferences.audioPanningQuality === "Low"  ? PanningModelType.EqualPower
            APP.store.state.preferences.audioPanningQuality === "High" ? PanningModelType.HRTF
            : null;
}
```

If we always add `panningModel` to `preferencesOverrides`, these layers can never affect the `panningModel`: `defaults`, `sceneOverrides`, `audioOverrides`, `audioDebugPanelOverrides`, `zoneSettings`.